src/HtmlSanitizer/HtmlSanitizer.csproj

```diff
@@ -44,14 +44,10 @@
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
Av någon anledning kom det in konstiga referenser så fick kopiera över de riktiga.